src/nanotron/config/config.py

```diff
@@ -416,8 +419,7 @@ def get_config_from_dict(
                 RecomputeGranularity: lambda x: RecomputeGranularity[x.upper()],
                 SamplerType: lambda x: SamplerType[x.upper()],
             },
-            # strict_unions_match=True,
-            strict=True,
+            strict_unions_match=True,
```

Comment:
why? are you sure this doesn't break anything?

Reply:
reverted it. I did some experiments and forgot about it.